nextflow/accession.nf

```diff
@@ -187,6 +187,8 @@ process csi_index_vcf {
 
     """
     cd $params.public_dir
+    # remove the uncompressed vcf file
+    rm *.vcf
```

Comment:
I'm still worried that we could be removing files that are in the `60_public` (even if they should not be).
Can't we get the `accessioned_filenames` and remove them by name ?

Reply:
fair enough, done!